db/schema.rb

```diff
@@ -433,18 +452,26 @@
   create_table "roles", force: :cascade do |t|
     t.string   "name",        null: false
     t.text     "description"
-    t.datetime "created_at"
-    t.datetime "updated_at"
+    t.datetime "created_at",  null: false
+    t.datetime "updated_at",  null: false
     t.string   "slug"
   end
 
   add_index "roles", ["slug"], name: "index_roles_on_slug", unique: true, using: :btree
 
+<<<<<<< HEAD
```

Comment:
There's another one further up, too. I see we run `rake db:migrate` before `rspec` in Travis, so `schema.rb` gets ignored. We should probably change that, somehow - run rspec once with the existing `schema.rb` and then check that `rake db:migrate` doesn't change it? I don't think the changes it makes to `schema.rb` are guaranteed to be idempotent, unfortunately :-(